src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-flex;
+  margin: 0 2px;
+  background-position: center;
+}
+
+.stars__star:first-of-type {
+  margin-left: 0;
+}
+
+.stars .stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star ,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}

```

```suggestion
.stars .stars__star {
  width: 16px;
  height: 16px;
  background-image: url(./images/star.svg);
  background-repeat: no-repeat;
}
```

you can move `background-image` to `.stars__star`

---

done